src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -9220,9 +9220,11 @@ def test_aks_create_with_azurekeyvaultkms_private_key_vault(
 
         create_cmd = (
             "aks create --resource-group={resource_group} --name={name} "
+            "--aks-custom-headers AKSHTTPCustomFeatures=Microsoft.ContainerService/EnableAPIServerVnetIntegrationPreview "
             "--assign-identity {identity_id} "
             "--enable-azure-keyvault-kms --azure-keyvault-kms-key-id={key_id} "
             "--azure-keyvault-kms-key-vault-network-access=Private --azure-keyvault-kms-key-vault-resource-id {kv_resource_id} "
+            "--enableVnetIntegration=true "
```

Comment:
The option name should be `--enable-apiserver-vnet-integration`

Reply:
Good catch!